main/store/migrations/index.js

```diff
@@ -871,7 +871,9 @@ module.exports = {
       .sort((a, b) => a - b)
       .forEach((version) => {
         if (parseInt(state.main._version) < version && version <= migrateToVersion) {
-          log.info(`Applying state migration: ${version}`)
+          if (process.env.NODE_ENV !== 'test') {
```

Comment:
would prefer to keep test-specific code out of the main code wherever possible. this does give me an idea though: what about a manual mock of the logger that we can use for this type of filtering and then probably other things as well, maybe getting rid of it from all the tests' `beforeAll()` hooks?

Reply:
Sounds good, yeah there's some other error logs which would be nice to remove at the same time...